src/utils/dateUtils.ts

```diff
@@ -0,0 +1,13 @@
+export const convertUtcToTimeDiff = (lastUpdatedTime: string): string => {
```

Comment:
I think this is out of scope of the current PR - but I'm not sure if we can fix this regardless, since the `lastUpdatedTime` is something that's provided by the Github API directly, and it doesn't really make sense to convert to epoch time on our backend and then convert it again to human readable text on our frontend

Reply:
Sure, this is reasonable - i don't think we want to brand the type as a `GithubDate` here (see [here](https://medium.com/@KevinBGreene/surviving-the-typescript-ecosystem-branding-and-type-tagging-6cf6e516523d) for more info on this) but we should add a JSDoc over this function to 
1. inform devs that this is a date/time string from github 
2. explicitly stating the format itself, so that if we do need to use this by passing in non-github dates, we don't break the implicit assumption of correct formatting

Reply:
8f897987